com.ibm.jbatch.container/META-INF/MANIFEST.MF

```diff
@@ -24,7 +24,7 @@ Import-Package: com.ibm.jbatch.spi,
  javax.enterprise.context;resolution:=optional,
  javax.enterprise.context.spi;resolution:=optional,
  javax.enterprise.inject;resolution:=optional,
- javax.enterprise.event;version="[1.0,2)";resolution:=optional,
+ javax.enterprise.event;version="[2.0,3)";resolution:=optional,

```

Comment:
I think a better change will be to use:

  `javax.enterprise.event;version="[1.0,3)";resolution:=optional,`

so the code can be kept in common across GF 4 and 5.      

Obviously that would be trivial for me to do myself, but I think it's better etiquette to ask you to make that change and resubmit if you want to (or let me know if you don't care).